doc/netplan-tutorial.md

```diff
@@ -0,0 +1,560 @@
+
+# Getting started with Netplan
```

Comment:
praise: very nice tutorial! I wonder if we should extend it for some simple bridge and/or bonding case? (As this is a typical server usecase)

Reply:
Thanks! I added one more example using a bond interface.